examples/example-app/build.sbt

```diff
@@ -0,0 +1,35 @@
+import com.typesafe.sbt.SbtMultiJvm.multiJvmSettings
+import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
+import org.virtuslab.ash.AkkaSerializationHelperPlugin
+
+name := "example-app"
+version := "0.1"
+scalaVersion := "2.13.6"
+
+val circeVersion = "0.14.2"
+val akkaVersion = "2.6.19"
+val logbackVersion = "1.2.11"
+
+lazy val `example-app` = project
+  .in (file("."))
+  .enablePlugins(AkkaSerializationHelperPlugin)
+  .settings(multiJvmSettings: _*)
+  .settings(
+    libraryDependencies ++= akkaDependencies ++ ashDependencies ++ Seq(logbackDependency, circeDependency),
+    fork :=  true,
```

Comment:
Maybe mention https://discuss.lightbend.com/t/akka-projection-getting-started-guide-example-could-not-run-eventgeneratorapp/9434 in a code  comment